### ISSUE TITLE
Add list of keyboard shortcuts

### ISSUE THREAD

**Issue by m1k3s0** [computing-den/unforget#12]
As someone who prefers keyboard navigation, it would be great to see the keyboard shortcuts referenced somewhere. Unless I missed it somewhere more obvious, I needed to look at the NotePage.tsx code to find the shortcuts. The 'n and '/' shortcuts particularly are very handy.

**Comment by computing-den** [comment]
Yeah I should add a help page but for now at least you don't need to look at the code. Instead, hover your mouse over the buttons and the tooltip should show you the shortcut.

**Comment by m1k3s0** [comment]
Ah, thank you. I didn't notice that.
